config.json

```diff
@@ -684,6 +684,64 @@
                }
             }
          ]
+      },
+      {
+         "url":"https://github.com/ebuzzing/lib-front-models.git",
+         "rebase":[
+            {
+               "from":"master",
+               "to":"feat.*",
```

Comment:
i'm just thinking about this, but it's a regexp :) so you can add a single rule because you have only one rebase "from" and same settings for the automatic rules

with a "to" like this -> `"^(feat|fix|improv)"` just one rule is sufficient ;)
and i have to do the same for all the config XD